howso/utilities/feature_attributes/pandas.py

```diff
@@ -527,9 +538,22 @@ def _infer_floating_point_attributes(self, feature_name: str) -> dict:
                         'of 64 bits.'
                     )
 
+        # Only override types left are ordinal and continuous
+        if feature_type_override:
+            attributes['type'] = feature_type_override
+
         return attributes
 
-    def _infer_datetime_attributes(self, feature_name: str) -> dict:
+    def _infer_datetime_attributes(
+        self,
+        feature_name: str,
+        feature_type_override: t.Optional[str] = None
+    ) -> dict:
+        if feature_type_override == 'nominal':
+            return {
```

Comment:
Do we not want the date_time_format and data_type set when nominal? Are nominal datetimes possible?

Reply:
Census seems to be that yes, we should still set the format. So this immediate return should be removed. Same with the date only variant.